addons/common/XEH_postInit.sqf

```diff
@@ -371,5 +371,23 @@ GVAR(deviceKeyCurrentIndex) = -1;
 {false},
 [0xC7, [true, false, false]], false] call cba_fnc_addKeybind;  //SHIFT + Home Key
 
+//Map opened/closed Events:
+GVAR(mapOpened) = false;
+[] spawn {
+    waitUntil {(!isNull findDisplay 12)};
```

Comment:
```
// This spawn is probably worth keeping, as pfh don't work natively on the briefing screen and IDK how reliable the hack we implemented for them is.
// The thread dies as soon as the mission start, so it's not really compiting for scheduler space.
```

Since that was written, PFEH now work natively on the briefing screen.  So we could convert all these spawns to PFEH.

But like it also says, these are "one and done", they will be gone from the scheduler once the game starts, so it really doesn't matter.